functions/go/set-namespace/examples/expected/deploy.yaml

```diff
@@ -0,0 +1,40 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# This file configures the hello-world app which serves public web traffic.
+apiVersion: apps/v1 # Copyright 2019 Google LLC
```

Comment:
There is a small issue that `# Copyright 2019 Google LLC` is supposed to be at the beginning of the License header. I suspect there is a bug in the underlying yaml library.